client/platform/desktop/backend/native/common.ts

```diff
@@ -830,10 +848,21 @@ async function _importTrackFile(
 
     const newPath = npath.join(projectDirAbsPath, npath.basename(newFileName));
 
-    await fs.copy(
-      userTrackFileAbsPath,
-      newPath,
-    );
+    if (jsonMeta.originalVideoFile && await nistSerializers.confirmNistFile(userTrackFileAbsPath)) {
```

Comment:
I'm confused by this line.  isnt `type` more appropriate than `originalVideoFile` for determining dataset type?

I also kinda don't like that we have a data type that only applies to video.   In the `annotationImport` function above, it doesn't look like you've added any code to enforce that the user must only import NIST data on videos.

Reply:
I was just looking at it and while I can enforce the type check on the `annotationImport` I think the right thing to do is to throw Proper error messages in the annotation import.  `annotationImport.vue` right now just looks for simple boolean value on import I think that should change on the desktop/server end to be wrapped in a try/catch and catch and display errors as to why it failed.  That might be better done in a different PR.

Reply:
Sorry I'm not following.

It seems to me like this line should not use `jsonMeta.originalVideoFile`.  Is there a reason you used that as a proxy for `jsonMeta.type`, and can it be changed?

Reply:
Yeah I'm sorry I wasn't clear there and jumped a few steps ahead.  The `originalVideoFile` was a mistake on my part.  I fixed that to use `jsonMeta.type`.  The second part of your comment about `annotationImport` not having any guards against applying on video I fixed that as well using the `jsonMeta.type` to prevent NIST loading on image-sequences. 

The `annotationImport` function returns `true` for imported and `false` for rejected.  When rejected it just displays a generic prompt saying "failed to import {X}".  I'm saying that this should be refactored on both the desktop/server side to throw descriptive errors and the client should display the error.  That way they can see that they can't import a NIST file on image-sequences.